### DISCUSSION TITLE
Bootstrap theme

### DISCUSSION THREAD

**Discussion by piernik** [shlomiassaf/ngrid#67] [New Features]
Hi. I'm very impressed with this component. It says that it's ui agnostic, so I'd like to make `bootstrap` theme for it.
I need to understand Your code and conception.
I forked Your code, but it's very complicated for me :) Maybe I'll come back to that. For now I've made my new project that only uses Your library. But to the case.

I see that Your theme is based on `@angular/material` scss files. So I need to use Your classes (`pbl-ngrid-x`) and style them to `bootstrap` theme - right?
Are there some `pbl` mixins I could use? Or maybe You have some guides to make it right?

**Comment by piernik** [comment]
Here is what I have so far:
![ngrid-bootstrap-1](https://user-images.githubusercontent.com/420691/70148431-71ed9800-16a6-11ea-929c-119fd0bf312e.png)

**Comment by shlomiassaf** [comment]
Hi @piernik 

Amazing!!!


This is great news, I'm looking forward for this collaboration!

I'm currently away on a business trip but I'll make up some time next week to dive deep into this.

Note that the theme itself is only the CSS part and does not require an actual package other then a means to distribute it.

The `@penula/ngrid-material` package includes the material theme but also additional components like a checkbox column, a sort UI component for the column headers, pagination etc... It uses angular material components internally to do that.

We should probably add these as well for bootstrap, cause their quite common.... probably using ng-bootstrap cause it's the most famous package...
